src/Analytics.php

```diff
@@ -568,10 +584,12 @@ protected function sendHit($methodName)
             throw new InvalidPayloadDataException();
         }
 
-        return $this->getHttpClient()->post(
-            $this->getUrl(),
-            $this->isAsyncRequest
-        );
+        if (!$this->isDisabled) {
```

Comment:
In here, I think we should return an AnalyticsResponse "Null Object" instead of just `null`